ui/src/main/resources/edu/wpi/grip/ui/MainWindow.fxml

```diff
@@ -217,6 +217,7 @@
         <fx:include source="Deploy.fxml" fx:id="deployPane" />
         <fx:include source="AboutDialog.fxml" fx:id="aboutPane" />
         <fx:include source="analysis/Analysis.fxml" fx:id="analysisPane" />
+        <fx:include source="codegeneration/CodegenDialog.fxml" fx:id="codegenOptionsPane"/>
```

Comment:
This main window controller feels like its doing too much...
Musing, may not be something you can change.